pkg/tools/sandbox/builder.go

```diff
@@ -43,40 +43,41 @@ import (
 )
 
 const (
-	defaultContextTimeout = time.Second * 15
-	// DefaultRegistryExpiryDuration is a default expiry duration for registry
-	DefaultRegistryExpiryDuration = 500 * time.Millisecond
+	defaultContextTimeout         = time.Second * 15
+	defaultRegistryExpiryDuration = time.Minute
```

Comment:
It is used once. Please use it inline.

Reply:
It definitely would be modified in future. So it should better stay as a constant to prevent us from future searches across the file :)

Reply:
Other things like DNSDomainName and so on also declared inline in NewBuilder

Reply:
Done.